tapiro/core/src/main/scala/io/buildo/tapiro/Util.scala

```diff
@@ -128,9 +157,10 @@ class Util() {
         Meta.packageFromList(`package`),
         requiredPackages.toSet.map(Meta.packageFromList),
         Term.Name(tapirEndpointsName),
-        Meta.codecsImplicits(routes),
-        routes.map(TapirMeta.routeToTapirEndpoint(Term.Name(tapirEndpointsName))),
-        routes.flatMap(TapirMeta.routeClassDeclarations),
+        Type.Param(List(), Name(authTokenName), List(), Type.Bounds(None, None), List(), List()),
```

Comment:
Maybe we can make this parameter more uniform between all three `create*Endpoints` functions, since I think all use it the same way? Either always as `Type.Param` or always as `Type.Name`.